DYDX-Bots/Approve_Spending_Monitor/README.md

```diff
@@ -0,0 +1,45 @@
+# Spending Approval monitor
+
+## Description
+
+This bot detects large spending approvals on dYdX Liquidity and Safety modules.
+
+> Large is set as a percentage of the total staked tokens for each module.
+> The percentage can be ajusted by updating `THRESHOLD_PERCENTAGE` in `utils.ts`.
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+- DYDX-19-1
+
+  - Fired when `Approval` event is emitted on Liquidity Module with a large `value`.
+  - Severity is always set to "Info".
+  - Type is always set to "Info".
+  - Metadata includes:
+    - `value`: amount of tokens that is approved to be spent.
+    - `owner`: address of the approved tokens owner.
+    - `spender`: address approved to spend the tokens.
+
+- DYDX-19-2
+
+  - Fired when `Approval` event is emitted on Safety Module with a large `value`.
+  - Severity is always set to "Info".
+  - Type is always set to "Info".
+  - Metadata includes:
+    - `value`: amount of tokens that is approved to be spent.
+    - `owner`: address of the approved tokens owner.
+    - `spender`: address approved to spend the tokens.
+
+## Test Data (Kovan testnet)
```

Comment:
Including a note to update `THRESHOLD_PERCENTAGE` in `util.ts` to run transactions below would be helpful.